check_imagestreams.py

```diff
@@ -41,6 +41,17 @@ def check_imagestreams(self):
             return 0
         for f in json_files:
             print(f"Checking file {str(f)}.")
+            # Get os_version from stream file name
+            try:
+                os_version = str(f.stem).split("-")[1]
+            except IndexError:
+                print(f"File {str(f)} does not contain version like centos7|rhel7|fedora.")
+                continue
+            exclude_file = p / self.version / f".exclude-{os_version}"
```

Comment:
Hm, nice, I didn't know this before, so I learnt something new thanks to this PR :)